Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -387,7 +387,7 @@ public void ResolveBorderElementB ()
     public void DropDownList ()
     {
       WebPadding padding;
-      if (Helper.BrowserConfiguration.IsChrome())
+      if (Helper.BrowserConfiguration.IsChromium())

```

Comment:
> It might be better to have Chrome, Edge, (and Chromium if we actually want to have a shortcut option)

That is the case currently. We do still offer the `BrowserConfigurationExtensions.IsChrome(...)` extension method, edge and chromium versions were added. The usage of `.IsChromium()` makes sense here, as there is no difference between Edge and Chrome for this test.